nautobot_golden_config/filters.py

```diff
@@ -466,4 +461,4 @@ class Meta:
         """Boilerplate filter Meta data for Config Plan."""
 
         model = models.ConfigPlan
-        fields = ["id", "device", "created", "plan_type", "feature", "change_control_id", "status"]
+        fields = ["id", "created", "change_control_id"]
```

Comment:
Same error as above:
![Screenshot 2023-09-13 at 2 02 58 PM](https://github.com/nautobot/nautobot-plugin-golden-config/assets/10467633/fbdcffaa-9752-4ff0-8e6a-2121881b7ad0)

Reply:
got a PR coming in for this